src/tcp.c

```diff
@@ -246,8 +251,14 @@ static int tcp_fsm(struct tcp_conn_tcb *conn,
             }
             rs->tcp_flags |= TCP_ACK;
             rs->tcp_ack_num = rs->tcp_seqno + 1;
-            srand(time(NULL));
```

Comment:
Why did you remove `srand`?

Reply:
`srand(time(NULL));`  was the seed of  `rand()%100;`, and I replaced both lines with kernel random number source device.
So `srand` is no longer needed.

Reply:
You have to take care of both POSIX (generic) and Linux kernel random number generator.

Reply:
Do you mean that using `srand` (previous generator) while the environment is POSIX?

Reply:
Exactly. Only the files in `linux/` directory can introduce Linux-specific implementations. Always keep POSIX compatibility in mind.